### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by slimreaper35**
# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] Docs updated (if applicable)
- [x] Docs links in the code are still valid (if docs were updated)

**Note:** if the contribution is external (not from an organization member), the CI
pipeline will not run automatically. After verifying that the CI is safe to run:

- [approve GitHub Actions workflows][approve-gh-actions] by clicking a button
- approve the Red Hat Trusted App Pipeline container build by commenting `/ok-to-test`
  (as is the [standard for Pipelines as Code][pac-running-pipeline])

[approve-gh-actions]: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks
[pac-running-pipeline]: https://pipelinesascode.com/docs/guide/running/#running-the-pipelinerun

### CODE REVIEW

**Review by eskultety — APPROVED** [review]

**Comment by slimreaper35** [comment]
Any other suggestions for .gitignore?

**Review by a-ovchinnikov — APPROVED** [review]
I would suggest rewording commit message slightly, to something along the lines of "Adding default cachi2 output locations to .gitignore to prevent them being accidentally commited.", but this is absolutely not a blocker (and arguably a matter of taste). LGTM!
